data/games_fixture.csv

```diff
@@ -0,0 +1,15 @@
+game_id,season,type,date_time,away_team_id,home_team_id,away_goals,home_goals,venue,venue_link
+2016030231,20162017,Postseason,4/27/17,18,19,2,3,Talen Energy Stadium,/api/v1/venues/null
+2016030232,20162017,Postseason,4/29/17,18,19,2,3,Talen Energy Stadium,/api/v1/venues/null
+2016030233,20162017,Postseason,4/30/17,19,18,1,3,Allianz Field,/api/v1/venues/null
+2016030234,20162017,Postseason,5/3/17,19,18,1,2,Allianz Field,/api/v1/venues/null
+2016030235,20162017,Postseason,5/6/17,18,19,1,2,Talen Energy Stadium,/api/v1/venues/null
+2016030236,20162017,Postseason,5/7/17,19,18,1,3,Allianz Field,/api/v1/venues/null
+2014030171,20142015,Postseason,4/17/15,52,24,2,2,Rio Tinto Stadium,/api/v1/venues/null
+2014030172,20142015,Postseason,4/19/15,52,24,1,2,Rio Tinto Stadium,/api/v1/venues/null
+2014030173,20142015,Postseason,4/21/15,24,52,3,2,Providence Park,/api/v1/venues/null
+2014030174,20142015,Postseason,4/23/15,24,52,3,2,Providence Park,/api/v1/venues/null
+2014020167,20142015,Regular Season,11/3/14,53,19,4,3,Talen Energy Stadium,/api/v1/venues/null
+2014021208,20142015,Regular Season,4/10/15,18,53,1,2,Mapfre Stadium,/api/v1/venues/null
+2014020354,20142015,Regular Season,11/30/14,53,52,2,1,Providence Park,/api/v1/venues/null
+2014021005,20142015,Regular Season,3/12/15,53,24,3,2,Rio Tinto Stadium,/api/v1/venues/null
```

Comment:
Hopefully this will be enough data that we can test everything we need to. We can add more data later if need be